docs/contributing/coding-standards/component-options.md

```diff
@@ -231,37 +69,12 @@ In our example, `rememberExpanded` becomes `data-remember-expanded`.
 
 ### In Nunjucks
 
-Unlike the `data-*` attribute in HTML and our use of `dataset` in JavaScript, there is no intrinsic link between the Nunjucks parameter name and the names used elsewhere. The Nunjucks parameter can therefore be named differently, if convenient.
+Unlike the `data-*` attribute in HTML, there is no intrinsic link between the Nunjucks parameter name and the names used elsewhere. The Nunjucks parameter can therefore be named differently, if convenient.
 
 A common case is specifying whether a parameter accepts HTML or only plain text. For example, if a configuration option's value is inserted into the page using `innerText`, you might want to name the Nunjucks parameter something like `sectionLabelText` to indicate that HTML will not be parsed if provided.
 
 There is more guidance on naming Nunjucks parameters in [the Nunjucks API documentation](https://github.com/alphagov/govuk-frontend/blob/main/docs/contributing/coding-standards/nunjucks-api.md#naming-options).
 
 ## Namespacing configuration options
 
-You can group configuration options in JavaScript and HTML together by using namespaces; period-separated strings that prefix the configuration option name. Namespaces follow the same formats as other option names, being camelCase in JavaScript and kebab-case in HTML.
-
-These are most commonly used for translation strings, which are usually namespaced under `i18n` (short for 'internationalisation').
-
-For example, we could namespace our `rememberExpanded` option under the `stateInfo` namespace. Our `data-*` attribute would now be named `data-state-info.remember-expanded` and accessed in the component's JavaScript using `this.config.stateInfo.rememberExpanded`.
-
-The `extractConfigByNamespace` JavaScript helper can be used to create an object containing _only_ the configuration options that belong to a certain namespace.
-
-```mjs
-import { mergeConfigs, extractConfigByNamespace, normaliseDataset } from '../../common/configuration.mjs'
-import { GOVUKFrontendComponent } from '../../govuk-frontend-component.mjs'
-
-export class Accordion extends GOVUKFrontendComponent {
-  constructor($root, config = {}) {
-    super($root)
-
-    this.config = mergeConfigs(
-      Accordion.defaults,
-      config,
-      normaliseDataset(Accordion, this.$root.dataset)
-    )
-
-    this.stateInfo = extractConfigByNamespace(Accordion, this.config, 'stateInfo');
-  }
-}
-```
+When passing configuration into your component's constructor, you can use nested objects to group options. This isn't possible when using data attributes, which only accept strings. Instead, you should follow our [naming conventions for data attributes](https://frontend.design-system.service.gov.uk/building-configurable-components/#receiving-configuration-from-data-attributes).
```

Comment:
**suggestion** The "This isn't possible" reads as if you won't be able to set nested options through data attributes. Maybe we can spin this more positively with:

```suggestion
When passing configuration into your component's constructor, you can use nested objects to group options. 

Data attributes only accept strings, but our  [naming conventions for data attributes](https://frontend.design-system.service.gov.uk/building-configurable-components/#receiving-configuration-from-data-attributes) allow you to set options in nested objects using a dot `.` separator for each level of nesting. For example `data-i18n.showLabel="Show"` will set the same option as the following constructor argument:

```json
{
  i18n: {
     showLabel: 'Show'
  }
}
```